polyglot_piranha.pyi

```diff
@@ -13,7 +13,7 @@ from __future__ import annotations
 from typing import List, Optional, Literal
 
 # Languages that Piranha supports (see ./src/models/language.rs)
-PiranhaLanguage = Literal["java", "kt", "kotlin", "go", "py", "swift", "ts", "tsx", "thrift", "strings", "scm", "scala", "rb", "yaml", "yml"]
+PiranhaLanguage = Literal["java", "kt", "kotlin", "go", "python", "swift", "typescript", "tsx", "thrift", "strings", "scm", "scala", "ruby", "yaml", "yml"]
```

Comment:
Quick question: is this a safe change? 

Looking at the names here it feels that we are probably also using this as file extensions? 

Having `kt` and `kotlin` here also seems to agree with that theory. Should we include `py` and `ts` here as well?

Reply:
a bit of the blue, but I added the this alias because it would otherwise break something. I don't remember exactly what